.travis.yml

```diff
@@ -24,7 +24,12 @@ matrix:
     # Latest xcode image (needs periodic updating)
     - os: osx
       compiler: clang
-      osx_image: xcode11.2
+      osx_image: xcode12.3
+      arch: amd64
+    # Oldest xcode version supported by Homebrew
+    - os: osx
+      compiler: clang
+      osx_image: xcode10.3
```

Comment:
Commit title does not match this (Given the comment I suspect the title is the one in error)

Reply:
I guess one additional snag is that usage of travis sounds like it is going to become expensive (see https://github.com/axboe/fio/pull/1157 for details).

Reply:
> Commit title does not match this (Given the comment I suspect the title is the one in error)

Right, I have fixed the commit title.

Reply:
> I guess one additional snag is that usage of travis sounds like it is going to become expensive (see #1157 for details).

So we will need to run OS/X builds on Appveyor?

Reply:
> So we will need to run OS/X builds on Appveyor?

https://github.com/axboe/fio/pull/1157 adds macOS builds via GitHub Actions. The bad thing is that older macOS builds will no longer be available but that was always one of Travis' strengths  (e.g. Appveyor only supports two macOS releases - https://www.appveyor.com/docs/macos-images-software/ ).